MOBILE_API.xml

```diff
@@ -2817,17 +2868,283 @@
+            <description>Include exterior lights located at the back of the vehicle. For example, license plate lights, reverse lights, cargo lights, bed lights an trailer assist lights.</description>
+        </element>
+        <element name="EXTERIOR_LEFT_LIGHTS" value="903">
+            <description>Include exterior lights located at the left side of the vehicle. For example, left puddle lights and spot lights.</description>
+        </element>
+        <element name="EXTERIOR_RIGHT_LIGHTS" value="904">
+            <description>Include exterior lights located at the right side of the vehicle. For example, right puddle lights and spot lights.</description>
+        </element>
+    </enum>
+
+    <enum name="LightStatus" since="5.0">
+        <element name="ON"/>
+        <element name="OFF"/>
+    </enum>
+
+    <struct name="LightCapabilities" since="5.0">
+        <param name="name" type="LightName" mandatory="true" />
+        <!-- Assuming light ON/OFF status is always available -->
+        <param name="densityAvailable" type="Boolean" mandatory="false">
+            <description>
+                Indicates if the light's density can be set remotely (similar to a dimmer).
+            </description>
+        </param>
+        <param name="rgbColorSpaceAvailable" type="Boolean" mandatory="false">
+            <description>
+                Indicates if the light's color can be set remotely by using the RGB color space.
+            </description>
+        </param>
+    </struct>
+
+    <struct name="LightControlCapabilities" since="5.0">
+        <param name="moduleName" type="String" maxlength="100" mandatory="true">
+            <description>
+                The short friendly name of the light control module.
+                It should not be used to identify a module by mobile application.
+            </description>
+        </param>
+        <param name="supportedLights" type="LightCapabilities" minsize="1" maxsize="100" array="true" mandatory="true">
+            <description> An array of available light names that are controllable. </description>
+        </param>
+    </struct>
+
+    <struct name="LightState" since="5.0">
+        <param name="id" type="LightName" mandatory="true">
+            <description>The name of a light or a group of lights. </description>
+        </param>
+        <param name="status" type="LightStatus" mandatory="true"/>
+        <param name="density" type="Float" minvalue="0" maxvalue="1" mandatory="false" />
+        <param name="color" type="RGBColor" mandatory="false" />
+    </struct>
+
+    <struct name="LightControlData" since="5.0">
+        <param name="lightState" type="LightState" mandatory="true" minsize="1" maxsize="100" array="true">
+            <description>An array of LightNames and their current or desired status. No change to the status of the LightNames that are not listed in the array.</description>
+        </param>
+    </struct>
+
+    <enum name="DisplayMode" since="5.0">
```

Comment:
Rearranging likely needed to be done because order matters when Core parses the XML.

Though I believe that this change was just for consistency, since the general order for each module appears to be:
1. Related data structs/enums
2. Module data struct
3. Related capabilities structs/enums
4. Module capabilities struct